_layouts/default.html

```diff
@@ -2,7 +2,7 @@
   <main id="main-content" class="grid-container">
   	{% include step-indicator.html %}
     {{ content }}
-    <a href="#back" id="backButton" class="usa-button usa-button--outline">Back</a>
+    <a href="#back" id="backButton" class="usa-button usa-button--outline">Back to questions</a><button class="usa-button ">File complaint</button>
```

Comment:
May switch this to another layout, or move the buttons to the markdown file since they're specific to the success page.